<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="devices" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="connectButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Find devices</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="deviceList">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="playlistPlaceholder" native="true">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>No device selected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="playlist" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListView" name="playlistView">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="player">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QSlider" name="progressSlider">
              <property name="tracking">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="progressLabel">
              <property name="text">
               <string>00:00</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="albumImage">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="trackTitle">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Track title</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="artistName">
                <property name="font">
                 <font>
                  <italic>true</italic>
                 </font>
                </property>
                <property name="text">
                 <string>Artist name</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::AutoText</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="playButton">
                <property name="text">
                 <string>Play / Pause</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QPushButton" name="prevButton">
                  <property name="text">
                   <string>Prev</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="nextButton">
                  <property name="text">
                   <string>Next</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="playmodeButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Play Mode</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Volume</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="volumeSlider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="tracking">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="volumeLabel">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="muteButton">
              <property name="text">
               <string>Mute</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="recButton">
              <property name="text">
               <string>Record</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionQt"/>
   </widget>
   <widget class="QMenu" name="menuFile_2">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPlaylist">
    <property name="title">
     <string>Playlist</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>Add Item</string>
     </property>
     <addaction name="actionFiles"/>
     <addaction name="actionURL"/>
     <addaction name="actionMic"/>
     <addaction name="actionPulse"/>
    </widget>
    <addaction name="menuAdd"/>
    <addaction name="actionClear"/>
   </widget>
   <addaction name="menuFile_2"/>
   <addaction name="menuFile"/>
   <addaction name="menuPlaylist"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSettings">
   <property name="icon">
    <iconset theme="configure">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionRemoveItem">
   <property name="icon">
    <iconset theme="delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionPlayItem">
   <property name="icon">
    <iconset theme="media-playback-start">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionPauseItem">
   <property name="icon">
    <iconset theme="media-playback-pause">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset theme="network-connect">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionFiles">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Files</string>
   </property>
  </action>
  <action name="actionURL">
   <property name="icon">
    <iconset theme="applications-internet">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>URL</string>
   </property>
  </action>
  <action name="actionMic">
   <property name="icon">
    <iconset theme="audio-input-microphone">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Mic</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset theme="delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionPulse">
   <property name="icon">
    <iconset theme="audio-speakers">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Audio capture</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
